jdaviz/configs/default/plugins/markers/markers.py

```diff
@@ -84,14 +90,14 @@ def __init__(self, *args, **kwargs):
         self.table.headers_avail = headers
         self.table.headers_visible = headers
         self.table._default_values_by_colname = self._default_table_values
+        self.table.items_per_page = 15
+        self.table.hide_items_per_page = True

```

Comment:
this doesn't seem to actually do anything, I would have expected the dropdown for number of rows to be hidden, but I still see it.  Or does this only affect the astropy table but not the widget?

Reply:
(and for added context, Jenn recommended removing the option to change how many rows are shown per page)

Reply:
ok, so maybe remove these lines if they don't do the trick and create a follow-up ticket to try to remove with css?